frontend/src/pages/Dashboard/components/RecordDrawer/RecordDrawer.module.scss

```
@@ -227,6 +227,16 @@
   color: var(--ink-2);
   font-size: 13px;
   line-height: 1.7;
+  // 브리핑과 메모는 문단을 나눠 옵니다. 기본값이면 개행이 공백으로 뭉개집니다.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)